app/models/log_ticket.rb

```diff
@@ -0,0 +1,23 @@
+class LogTicket < ActiveRecord::Base
+  enum backend: [:s3, :local]
```

Comment:
> We shouldn't ever have both in any one environment.

Not sure about that, specially on staging we could test both. Also if we ever need to fix something  manually, we could use local FS easily to do that.